src/csharp/Grpc.IntegrationTesting.QpsWorker/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Reflection;
```

Comment:
Turns out any of my AssemblyInfo.cs files doesn't contain the copyright comment so I'd prefer to add them in a separate pull requests. Also AssemblyInfo.cs doesn't contain any actual code, so I think it's fine to address this later.